docs/project/contributors.md

```diff
@@ -114,163 +80,49 @@ Blocknet is an open-source project with a team consisting of contributors from a
-	<div class="img-round"><img src="/img/contributors/torben-berger.jpg"></div>
-	<div class="name">Torben Berger&nbsp;&nbsp;
-		<a href="https://www.linkedin.com/in/torben-berger-8a01a0156/" target="_blank"><i class="fa fa-linkedin-square fa-1x" aria-hidden="true"></i></a>
-	</div>
-	<div class="title"><a href="http://theblocknet.de/" target="_blank">theblocknet.de</a></div>
-	<div class="platform">Discord <span class="handle">@lautrer#7464</span></div>
-</div> -->
-
 <div class="contributor">
 	<div class="fa fa-user-circle" aria-hidden="true"></div>
-	<div class="name">@mushoz</div>
-	<div class="title">Telegram Moderator</div>
-	<div class="platform">Telegram <span class="handle">@jaapje</span></div>
-</div>
-
-<!-- <div class="contributor">
-	<div class="img-round"><img src="/img/contributors/arturo-martin.jpg"></div>
-	<div class="name">Arturo Martin
-		<a href="https://www.linkedin.com/in/arturomartindenicolas/" target="_blank"><i class="fa fa-linkedin-square fa-1x" aria-hidden="true"></i></a>
-	</div>
-	<div class="title">Development</div>
-	<div class="platform">&nbsp;<span class="handle"></span></div>
-</div> -->
-
-<!-- <div class="contributor">
-	<div class="img-round"><img src="/img/contributors/daniel-zorin.jpg"></div>
-	<div class="name">Daniel Zorin
-		<a href="https://www.linkedin.com/in/daniel-zorin-a9704647/" target="_blank"><i class="fa fa-linkedin-square fa-1x" aria-hidden="true"></i></a>
+	<div class="name">@littleeagle&nbsp;&nbsp;
+		<a href="https://twitter.com/littleeagle20" target="_blank"><i class="fa fa-twitter-square fa-1x" aria-hidden="true"></i></a>
 	</div>
-	<div class="title">Development</div>
-	<div class="platform">Discord <span class="handle">@jaun</span></div>
-</div> -->
-
-<!-- <div class="contributor">
-	<div class="fa fa-user-circle" aria-hidden="true"></div>
-	<div class="name">Vladimir</div>
-	<div class="title">Development</div>
-	<div class="platform">Discord <span class="handle">@corvaxx</span></div>
-</div> -->
-
-<!-- <div class="contributor">
-	<div class="img-round"><img src="/img/contributors/boa.jpg"></div>
-	<div class="name">@boa</div>
-	<div class="title">Development</div>
-	<div class="platform">Discord <span class="handle">@boa</span></div>
-</div> -->
+	<div class="title"><a href="https://blockdx.co/" target="_blank">blockdx.co</a></div>
+	<div class="platform">Discord <span class="handle">@littleeagle#3663</span></div>
+</div>
 
-<!-- <div class="contributor">
-	<div class="fa fa-user-circle" aria-hidden="true"></div>
-	<div class="name">Andrey Lazarko</div>
-	<div class="title">Development</div>
-	<div class="platform">Discord <span class="handle">@xenix</span></div>
-</div> -->
+<div class="contributor"> 
+	<div class="img-round"><img src="/img/contributors/nikita-chisnikov.jpg"></div>
+	<div class="name">Nikita Chisnikov</div>
+	<div class="title">Bridge Bot</div>
+	<div class="platform">Discord <span class="handle">@nik_chis#7731</span></div>
+</div>
 
 </div>
```

Comment:
Do you think that the following people should be included? - 

Tryou
Coinwhisperer
Conan
Superbayfan
Amishbatman
Crypto-dev
Floyd1256
Uzrus

Reply:
Good suggestions. Shorn & feel myself and Tryou should be added. Some of the others maybe a little later, after they've been with us a little longer and/or started submitting SB proposals, etc...